PySimpleGUI.py

```diff
@@ -2770,18 +2770,14 @@ def AddMenuItem(top_menu, sub_menu_info, element, is_sub_menu=False, skip=False)
     if type(sub_menu_info) is str:
         if not is_sub_menu and not skip:
             # print(f'Adding command {sub_menu_info}')
-            pos = sub_menu_info.find('_&')
+            pos = sub_menu_info.find('&')
             if pos != -1:
-                _ = sub_menu_info[:pos]
```

Comment:
I'm not a fan of using '_' except for very special circumstances.

Is there any particular reason you chose to use that name in this case?

It'll be the first and only use of _ in this entire project..

Reply:
The rewrite fixes the issue. No use of _ variables needed.

Reply:
"The rewrite fixes the issue. No use of _ variables needed."

I don't know what that means

Reply:
This is like the blind leading the blind... I'm no GitHub expert either. I'd say go to my fork on the browser: [venim/PySimpleGUI](https://github.com/venim/PySimpleGUI/blob/master/PySimpleGUI.py)

Reply:
I gave up and made the changes by hand.  I added the set_index feature you wanted on combo_box but did it by adding a parameter.  I have not tested all this yet, so I'm only checking into the Dev branch. 

I had no idea what this Pull process was like.  I can only hope the next dozen lines don't take another 2 hours.

Please check what I've done to see if it meets your liking.